docs/request-context.md

```diff
@@ -19,21 +19,37 @@ Orka by default exposes a Request Context to the app.
 
 It can be used for setting/getting variables that can be accessible in every code file (within the request's life). It uses nodeJS `async_hooks` to store this information.
 
-By default it appends `requestId`, `visitor` (if you have the option `config.visitor.cookie`) in the request context and `ddSpan`(if you have DataDog enabled). The `requestId` is retrieved using the option `config.traceHeaderName`
+By default it appends in the request context: `requestId`, `visitor` (if you have the option `config.visitor.cookie`), `ddSpan`(if you have DataDog enabled),
+and `istio-headers` that holds [Istio trace context headers](https://istio.io/latest/docs/tasks/observability/distributed-tracing/overview/#trace-context-propagation) (if you have the option `config.requestContext.istioTraceContextHeaders.enabled`).
+The `requestId` is retrieved using the option `config.traceHeaderName`
 
 ### Log Tracer
 
 If the Request Context is enabled, orka by default appends `requestId`, `correlationId` and `visitor` to all the application logs automatically.
 
+### Istio trace context headers
+
+If option `config.requestContext.istioTraceContextHeaders` is enabled, orka propagates [Istio trace context headers](https://istio.io/latest/docs/tasks/observability/distributed-tracing/overview/#trace-context-propagation)
+from incoming http requests to any outbound http request.
```

Comment:
It is enabled by default right?

Reply:
it's enabled by default, as long as async local storage is supported by nodejs
https://github.com/Workable/orka/blob/b784bcf3d47fa36b67cdfe69ddc91113d44ce83f/src/initializers/diamorphosis.ts#L102